app/scripts/models/auth_brokers/base.js

```diff
@@ -286,6 +286,10 @@ define([
        */
       emailVerificationMarketingSnippet: true,
       /**
+       * Should the view handle signed-in notifications from other tabs?
+       */
+      signedInNotificationHandler: true,
```

Comment:
You're right that we can do better though. How about `handleSignedInNotification`? Or `actOnSignedInNotification`? Or `synchroniseSignedInState`?